server/db/driver/pg/tables.go

```diff
@@ -206,7 +206,9 @@ func prepareMarkets(db *sql.DB, mktConfig []*dex.MarketInfo) ([]string, error) {
 				if err != nil {
 					return nil, fmt.Errorf("unable to update lot size for %s: %w", mkt.Name, err)
 				}
-				purgeMarkets = append(purgeMarkets, mkt.Name)
+				// archiver.markets use market schema name.
+				schema := marketSchema(mkt.Name)
+				purgeMarkets = append(purgeMarkets, schema)
```

Comment:
Wow good catch. Can you do a cursory audit of some of the other pg code to ensure we're not taking the market name directly anywhere else?

Reply:
Sure I can. Should I look all methods which have a call to `db.Exec`?

Reply:
Anywhere really. Query, Exec, helper functions that feed into other query methods, etc.